workspaces/ballerina/ballerina-visualizer/src/PopupPanel.tsx

```
@@ -96,6 +95,43 @@ const PopupPanel = (props: PopupPanelProps) => {
                         );
                     });
                     break;
+                case MACHINE_VIEW.ConnectionConfiguration:
+                    const connectorMetadata = machineState.metadata ?? {};
```

Biome (2.4.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)